### ISSUE TITLE
Add support for Emacs packages

### ISSUE THREAD

**Issue by breyno127** [OctoLinker/browser-extension#212]
This should be relatives straightforward. Most emacs dependencies are declared in a [`Cask`](https://github.com/cask/cask) file, `require`'d directly in a file with a `.el` extension or declared with [`use-package`](https://github.com/jwiegley/use-package)

This should be pretty straightforward and similar in implementation to [when `vimrc` support was added. ](https://github.com/OctoLinker/browser-extension/commit/7e21743).

I think I _should_ be able to handle it if not one opposes me taking a crack at it.

**Comment by benjamreynolds** [comment]
Wanted to check in on this. The main problem here is that MELPA does not have an API that we can hit to get the correct repo to redirect to. See melpa/melpa#2850. There is a bit of a workaround by hitting in [archives.json](https://melpa.org/archive.json) but the file is pretty large (Nearly a full Megabyte) which may be a little much to handle with [live-resolver](https://github.com/OctoLinker/live-resolver). Im curious how @stefanbuck and @josephfrazier feel about this approach, and if they think it would be feasible to do this with live-resolver.

**Comment by josephfrazier** [comment]
Thanks for following up on this. I think we could start off by simply having the live-resolver hit archives.json on every request. Then, if performance/bandwidth is an issue, or if we want to be nicer to melpa.org, we could add in caching based on the `Last-Modified` header.

I'll see if I can get this working on the live-resolver. If you've started work on this repo, feel free to open a PR, even if it's just work-in-progress so far.

**Comment by benjamreynolds** [comment]
I actually started my work by looking at how to get the live-resolver to work with MELPA. Ill start work on this with the assumption that we will get live-resolver working with archive.json.

**Comment by josephfrazier** [comment]
Oops, sorry for the duplicate work! I put together a MELPA resolver at https://github.com/OctoLinker/live-resolver/pull/41, if you want to take a look.

**Comment by benjamreynolds** [comment]
AWESOME! now that the live-resolver for Melpa is up, Ill take a crack at the browser extension work when I get some time this week. It shouldnt be too difficult.

**Comment by josephfrazier** [comment]
Yup, I decided to go ahead and merge https://github.com/OctoLinker/live-resolver/pull/41 to make it easier for to you work on this, without having to run a local instance of live-resolver. I look forward to seeing how it goes! As I mentioned earlier, feel free to open a PR as soon as you'd like, even if it doesn't work yet.
